cirq/protocols/json_serialization.py

```diff
@@ -421,6 +425,19 @@ def _cirq_object_hook(d, resolvers: Sequence[JsonResolver]):
             "Could not resolve type '{}' during deserialization".format(d['cirq_type'])
         )
 
+    if cls == _SerializedKey:
+        read_from_context = getattr(cls, 'read_from_context', lambda x, y: None)
```

Comment:
these could be simplified to `return _SerializedKey.read_from_context(context_map, **d)`, right? Why are we not sure that the `_SerializedKey` privte class will have a method?

Reply:
This is another mypy-wrangling line: `cls == _SerializedKey` is apparently insufficient for mypy to understand that `cls.read_from_context` is a valid function.

The alternative is to call `_SerializedKey.read_from_context` directly, which is what I'm opting for along with removing these types from the default resolver.